rig/routing_table/errors.py

```diff
@@ -0,0 +1,16 @@
+class MinimisationFailedError(Exception):
```

Comment:
I realise it may be the wrong convention but elsewhere in rig files containing just exceptions have been called "exceptions.py". Might be worth keeping things consistent here if you don't mind?